.github/workflows/x-keycloak-nodejs-admin.yml

```diff
@@ -14,21 +13,42 @@ on:
         required: true
       GH_TOKEN:
         required: true
-
 jobs:
   release:
     runs-on: ubuntu-latest
-
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v3.0.0
+        uses: actions/checkout@v3
         with:
-          repository: '${{ secrets.GH_ORG }}/keycloak-nodejs-admin-client'
+          repository: ${{ secrets.GH_ORG }}/keycloak-nodejs-admin-client
           token: ${{ secrets.GH_TOKEN }}
-          path: keycloak-nodejs-admin-client
           ref: ${{ inputs.tag }}
 
-      - name: Placeholder
-        working-directory: ./keycloak-nodejs-admin-client
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          check-latest: true
+          cache: npm
+
+      - name: Cache dependencies
+        id: cache-dependencies
+        uses: actions/cache@v3
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-${{ hashFiles('**/package-lock.json') }}
+
+      - name: Install dependencies
+        if: steps.cache-dependencies.outputs.cache-hit != 'true'
+        run: npm ci
+
+      - name: Create package tarball
         run: |
-          echo 'Hello world!'
+          npm run prepublishOnly
+          npm pack
+          for f in *.tgz; do mv "$f" keycloak-nodejs-admin-client.tgz; done
```

Comment:
If I'm reading that mv command right, you're intending to find the single tgz with an unneeded version string(?) included, and rename it to a standard filename?

I think that could be more succinctly written as this:

```
mv -T *.tgz keycloak-nodejs-admin-client.tgz
```

This form of the command would also produce an error if there are multiple tgzs for some reason, rather than silently picking the last one it sees, which could lead to a weird bug in the future.

Reply:
Good suggestion! I added a commit that includes this change.